compute/etc/sql_exporter/installed_extensions.sql

```diff
@@ -0,0 +1,32 @@
+WITH
+    dbs AS (
+        SELECT
+            datname
+        FROM
+            pg_catalog.pg_database
+        WHERE
+            datallowconn
+            AND datconnlimit <> - 2
+        LIMIT
+            500
+    )
+SELECT
+    t.extname,
+    t.extversion,
+    (t.extowner = 10) AS owned_by_superuser
+    COUNT(DISTINCT d.datname) AS n_databases,
+FROM
+    dbs d,
+    LATERAL (
+        SELECT
+            *
+        FROM
+            dblink (
```

Comment:
AFAIK, dblink establishes a normal connection to Postgres, so it will count as activity anyway

Also, sql_exporter runs all these queries at scraping time, so I can easily see that with many databases it will time out, and many could even start under 100 or something, depends on the scraping timeout and how loaded instance is. We already see scraping timeouts on some loaded computes